core/web/apiv2/auth.py

```diff
@@ -93,11 +84,25 @@ async def get_current_user(
     user = UserSensitive.find(username=username)
     if user is None:
         raise credentials_exception
-    if not user.enabled:
-        raise disabled_exception
+    request.state.username = user.username
     return user
 
 
+async def get_current_active_user(
+    current_user: Annotated[User, Security(get_current_user)]
+):
+    
+    disabled_exception = HTTPException(
+        status_code=status.HTTP_401_UNAUTHORIZED,
+        detail="User account disabled. Please contact your server admin.",
+        headers={"WWW-Authenticate": "Bearer"},
+    )
+
+    if not current_user.enabled:
+        raise disabled_exception
```

Comment:
Inline the exception here

Reply:
Fixed in 130729f5fb01ad9fa911fc923f75619ba586e741